app/views/text_components/_form.html.haml

```diff
@@ -1,133 +1,125 @@
-                %option{disabled: true, selected: true} Please select a sensor
-                - @sensors.each do |sensor|
-                  %option{data: {editor: {'markup-command': 'text', 'markup-text': "{value(#{sensor.id})}"}}}
-                    = sensor.name
-            .text-editor__toolbar__item
-              Events
-              %select{'data-editor-markup-select': true}
-                %option{disabled: true, selected: true} Please select an event
-                - @events.each do |event|
-                  %option{data: {editor: {'markup-command': 'text', 'markup-text': "{date(#{event.id})}"}}}
-                    = event.name
-
-          .text-editor__fields
-            .field.text-editor__field
-              .field__label Heading
-              = f.input :heading, as: :text, :label => false, input_html: {class: 'field__input'}
-            .field.text-editor__field
-              .field__label Introduction
-              = f.input :introduction, as: :text, :label => false, input_html: {class: 'field__input'}
-            .field.text-editor__field
-              .field__label Main Part
-              .field__count
-              = f.input :main_part, as: :text, :label => false, input_html: {class: 'field__input field__input--large'}
-            .field.text-editor__field
-              .field__label Closing
-              = f.input :closing, as: :text, :label => false, input_html: {class: 'field__input'}
-
-
-    .form__section
-
-      .form__section__header
-        %h4.step
-          .step__counter 4
-          Chatbot Q/A
-          .step__action
-            %a.form__section__action--next.btn.btn-sm.btn-primary{href: '#', 'data-form-toggle': 'next-section'} Continue
-            %a.form__section__action--open.btn.btn-sm.btn-secondary{href: '#', 'data-form-toggle': 'open-section'} Edit
-          
-      .form__section__body
-        .qa
-          = f.simple_fields_for :question_answers do |question_answer|
-            = render 'question_answers/fields', f: question_answer
-
-          = link_to_add_association button_tag("Add Question & Answer", type: 'button', class: 'btn btn-secondary'), f, :question_answers, partial: 'question_answers/fields.html.haml'
-
-    .form__section
-
-      .form__section__header
-        %h4.step
-          .step__counter 5
-          Output
-          .step__action
-            %a.form__section__action--open.btn.btn-sm.btn-secondary{href: '#', 'data-form-toggle': 'open-section'} Edit
-
-      .form__section__body
-
-        = f.association :channels, collection: Channel.order('name'), selected: text_component.channels.count > 0 ? text_component.channel_ids : Channel.find_by({name: 'sensorstory'}).id, input_html: {'data-choices': true}
-
-
-  .form-actions
-    = f.button :submit, class: 'btn btn-primary'
+.container
+  = simple_form_for([@report, @text_component]) do |f|
+    = f.error_notification
+    .form
+      .row.form__section
```

Comment:
Not necessary. I thought I could fix phantomjs with it. Turns out no.

Try out phantom js and replace the `click_regardless_of_overlapping_element`. You will see that the edit buttons are misaligned in the screenshot. I hoped I could fix it with explicit grid. Or with a smaller screen width (force stacked layout). None of those solutions work and in the end I had to use `.trigger('click')` which is only available in `:poltergeist`. Bad.